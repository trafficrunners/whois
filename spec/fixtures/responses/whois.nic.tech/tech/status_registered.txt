Domain Name: GOOGLE.TECH
Domain ID: D9157622-CNIC
WHOIS Server: whois.markmonitor.com
Referral URL: https://www.markmonitor.com/
Updated Date: 2015-08-04T09:00:43.0Z
Creation Date: 2015-07-29T00:00:00.0Z
Registry Expiry Date: 2017-07-29T23:59:59.0Z
Sponsoring Registrar: MarkMonitor, Inc (TLDs)
Sponsoring Registrar IANA ID: 292
Domain Status: serverTransferProhibited https://icann.org/epp#serverTransferProhibited
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Domain Status: clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited
Domain Status: clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited
Registrant ID: mmr-87489
Registrant Name: DNS Admin
Registrant Organization: Google Inc.
Registrant Street: 1600 Amphitheatre Parkway
Registrant City: Mountain View
Registrant State/Province: CA
Registrant Postal Code: 94043
Registrant Country: US
Registrant Phone: +1.6502530000
Registrant Phone Ext:
Registrant Fax: +1.6502530001
Registrant Fax Ext:
Registrant Email: dns-admin@google.com
Admin ID: mmr-87489
Admin Name: DNS Admin
Admin Organization: Google Inc.
Admin Street: 1600 Amphitheatre Parkway
Admin City: Mountain View
Admin State/Province: CA
Admin Postal Code: 94043
Admin Country: US
Admin Phone: +1.6502530000
Admin Phone Ext:
Admin Fax: +1.6502530001
Admin Fax Ext:
Admin Email: dns-admin@google.com
Tech ID: mmr-87489
Tech Name: DNS Admin
Tech Organization: Google Inc.
Tech Street: 1600 Amphitheatre Parkway
Tech City: Mountain View
Tech State/Province: CA
Tech Postal Code: 94043
Tech Country: US
Tech Phone: +1.6502530000
Tech Phone Ext:
Tech Fax: +1.6502530001
Tech Fax Ext:
Tech Email: dns-admin@google.com
Name Server: NS2.GOOGLE.COM
Name Server: NS4.GOOGLE.COM
Name Server: NS3.GOOGLE.COM
Name Server: NS1.GOOGLE.COM
DNSSEC: unsigned
>>> Last update of WHOIS database: 2015-09-18T00:34:52.0Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

This whois service is provided by CentralNic Ltd and only contains
information pertaining to Internet domain names registered by our
our customers. By using this service you are agreeing (1) not to use any
information presented here for any purpose other than determining
ownership of domain names, (2) not to store or reproduce this data in
any way, (3) not to use any high-volume, automated, electronic processes
to obtain data from this service. Abuse of this service is monitored and
actions in contravention of these terms will result in being permanently
blacklisted. All data is (c) CentralNic Ltd https://www.centralnic.com/

Access to the whois service is rate limited. For more information, please
see https://registrar-console.centralnic.com/pub/whois_guidance.

