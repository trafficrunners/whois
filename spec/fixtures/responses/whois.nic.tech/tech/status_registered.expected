#disclaimer
  %s == "Access to the whois service is rate limited. For more information, please see https://registrar-console.centralnic.com/pub/whois_guidance."


#domain
  %s == "google.tech"

#domain_id
  %s == "D9157622-CNIC"


#status
  %s == ["serverTransferProhibited https://icann.org/epp#serverTransferProhibited", "clientTransferProhibited https://icann.org/epp#clientTransferProhibited", "clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited", "clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited"]

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2015-07-29 00:00:00 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2015-08-04 09:00:43 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2017-07-29 23:59:59 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "292"
  %s.name         == "MarkMonitor, Inc (TLDs)"
  %s.organization == nil
  %s.url          == nil

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == "mmr-87489"
  %s[0].name          == "DNS Admin"
  %s[0].organization  == "Google Inc."
  %s[0].address       == "1600 Amphitheatre Parkway"
  %s[0].city          == "Mountain View"
  %s[0].zip           == "94043"
  %s[0].state         == "CA"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.6502530000"
  %s[0].fax           == "+1.6502530001"
  %s[0].email         == "dns-admin@google.com"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "mmr-87489"
  %s[0].name          == "DNS Admin"
  %s[0].organization  == "Google Inc."
  %s[0].address       == "1600 Amphitheatre Parkway"
  %s[0].city          == "Mountain View"
  %s[0].zip           == "94043"
  %s[0].state         == "CA"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.6502530000"
  %s[0].fax           == "+1.6502530001"
  %s[0].email         == "dns-admin@google.com"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[0].id            == "mmr-87489"
  %s[0].name          == "DNS Admin"
  %s[0].organization  == "Google Inc."
  %s[0].address       == "1600 Amphitheatre Parkway"
  %s[0].city          == "Mountain View"
  %s[0].zip           == "94043"
  %s[0].state         == "CA"
  %s[0].country       == nil
  %s[0].country_code  == "US"
  %s[0].phone         == "+1.6502530000"
  %s[0].fax           == "+1.6502530001"
  %s[0].email         == "dns-admin@google.com"


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns2.google.com"
  %s[0].ipv4 == nil
  %s[0].ipv6 == nil
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns4.google.com"
  %s[1].ipv4 == nil
  %s[1].ipv6 == nil
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.google.com"
  %s[2].ipv4 == nil
  %s[2].ipv6 == nil
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns1.google.com"
  %s[3].ipv4 == nil
  %s[3].ipv6 == nil
